lib/es5-helpers.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const ChildProcess = require('child_process')
+const Path = require('path')
+const FS = require('fs')
+
+let prefixPath = null
+
+function find(startDir, names) {
```

Comment:
Sounds good.  What is the required action here then?  Do you want to just commit whatever changes you want onto this branch, since I'm not sure how the helper should be used?